apiserver/facades/agent/provisioner/provisioner.go

```diff
@@ -871,6 +871,7 @@ func (ctx *prepareOrGetContext) ProcessOneContainer(env environs.Environ, idx in
 				info.CIDR = parentDeviceSubnet.CIDR()
 				info.ProviderSubnetId = parentDeviceSubnet.ProviderId()
 				info.VLANTag = parentDeviceSubnet.VLANTag()
+				info.IsDefaultGateway = firstAddress.IsDefaultGateway()
```

Comment:
I guess the presumption is that if we're allocating addresses via DHCP, that will set our gateway as well?

Reply:
Yes.